spydra/src/main/java/com/spotify/spydra/submitter/runner/SubmissionCliParser.java

```diff
@@ -17,22 +17,20 @@
 
 package com.spotify.spydra.submitter.runner;
 
+import static java.lang.Integer.min;
+
 import com.spotify.spydra.model.JsonHelper;
 import com.spotify.spydra.model.SpydraArgument;
-
-import org.apache.commons.cli.CommandLine;
-import org.apache.commons.cli.DefaultParser;
-import org.apache.commons.cli.HelpFormatter;
-import org.apache.commons.cli.Options;
-import org.apache.commons.lang3.StringUtils;
-
 import java.io.FileInputStream;
 import java.io.IOException;
 import java.util.LinkedList;
 import java.util.List;
 import java.util.UUID;
-
-import static java.lang.Integer.min;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
```

Comment:
Please auto-format in a separate commit. I believe the rest of the code spaces library imports from jdk imports.

Reply:
Sorry about that, intellij likes to automatically optimize imports. It is configured with the latest google style though.

Reply:
moved the import order change to a separate PR #45 and rebased this PR on that.